core/models/configuration.rb

```diff
@@ -117,6 +117,14 @@ def self.ssh_user_path(configuration_path)
     File.join(File.expand_path(configuration_path), 'ssh_user.json')
   end
 
+  def self.ssh_user_file(configuration_path, node_name)
+    File.join(File.expand_path(configuration_path), "#{node_name}_ssh_user.json")
```

Comment:
Can you put this configuration file inside the configuration directory, not outside it?